test/code-fixtures/projects/sfge-working-app/force-app/main/default/classes/UnusedInterface.cls

```diff
@@ -0,0 +1,4 @@
+public interface UnusedInterface {
```

Comment:
This interface will violate the `UnimplementedTypeRule` if the Code Analyzer is run against `test/code-fixtures/projects/sfge-working-app/force-app/main/default/classes`.